pt-BR/mail/chrome/messenger/addressbook/abResultsPaneOverlay.dtd

```diff
@@ -68,13 +68,13 @@
 <!ENTITY sortDescending.accesskey              "r">
 
 <!-- context menu -->
-<!ENTITY composeEmail.label                    "Compose Email To">
+<!ENTITY composeEmail.label                    "Criar email para">
 <!ENTITY composeEmail.accesskey                "C">
-<!ENTITY deleteAddrBookCard.label              "Delete">
+<!ENTITY deleteAddrBookCard.label              "Excluir">
 <!ENTITY deleteAddrBookCard.accesskey          "D">
-<!ENTITY newAddrBookCard.label                 "New Card...">
+<!ENTITY newAddrBookCard.label                 "Novo cartão...">
 <!ENTITY newAddrBookCard.accesskey             "N">
-<!ENTITY newAddrBookMailingList.label          "New Mailing List...">
+<!ENTITY newAddrBookMailingList.label          "Nova lista de emails...">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY newAddrBookMailingList.label          "Nova lista de destinatários...">